### PR TITLE
Add prerequisite to add-dag-tags documentation

### PR DESCRIPTION
**Pull request by tiago-palma**
Add a UI prerequisite section explaining that RBAC needs to be enabled.

### CODE REVIEW

**Review by potiuk — APPROVED** [review]

**Comment by potiuk** [comment]
This is cool, but can you make this change to master as well before we merge it please?  The process we have that people usually add the changes to master and cherry-pick them to 1.10 afterwards. But it's ok also to have two parallel PRs.

**Comment by potiuk** [comment]
(BTW in master there is no need to mention RBAC as this is the only option).

**Comment by tiago-palma** [comment]
> (BTW in master there is no need to mention RBAC as this is the only option).

In master these files already exist, the only difference is that it does not have the Prerequisite section

**Comment by potiuk** [comment]
Ah I see ! cool!

**Comment by potiuk** [comment]
I think there is a consistent problem that we hit with the tests. I am going to cherry-pick my changes for master soon to fix those. Would you mind waiting a couple of days before I do that and rebase ? (Monday it shoudl be ready).

**Comment by tiago-palma** [comment]
> I think there is a consistent problem that we hit with the tests. I am going to cherry-pick my changes for master soon to fix those. Would you mind waiting a couple of days before I do that and rebase ? (Monday it shoudl be ready).

Sure, not rush from my side. Thank you

File: docs/howto/add-dag-tags.rst [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+
+**Screenshot**:
+
+.. image:: ../img/add-dag-tags.png
+
+**UI Prerequisite**:
+
+The web UI with RBAC feature needs to be enabled; otherwise, the UI
+will not allow you to filter DAGs using tags nor show you the tag
+labels.
```
Comment (ashb) [inline docs/howto/add-dag-tags.rst:49]: For 2.0 (which is where the master branch is going to be released as) this section isn't needed as the RBAC UI is the only one that exists

```suggestion
```

**Review by ashb — CHANGES_REQUESTED** [review]

File: docs/howto/add-dag-tags.rst (hunk 1 above)
Comment (tiago-palma) [reply docs/howto/add-dag-tags.rst:49]: The idea was to have this prerequisite section for the documentation >=1.10.8 and <2.0 for those who do not have the latest installed.
For documentation >= 2.0, the prerequisite section can simply disappear?

Comment (mik-laj) [reply docs/howto/add-dag-tags.rst:49]: > In master these files already exist, the only difference is that it does not have the Prerequisite section

https://github.com/apache/airflow/pull/10852#issuecomment-690094813

**Pull request closed without merging**